app/preferences_ios.m

```diff
@@ -1,26 +1,23 @@
 // +build darwin
 // +build ios
 
+#import <stdbool.h>
 #import <Foundation/Foundation.h>
 
-int getPreferenceBool(const char* key, int fallback) {
+bool getPreferenceBool(const char* key, bool fallback) {
     NSString *nskey = [NSString stringWithUTF8String:key];
     id obj = [[NSUserDefaults standardUserDefaults] objectForKey:nskey];
 
     if (obj == nil) {
         return fallback;
     } else {
-        if ([[NSUserDefaults standardUserDefaults] boolForKey:nskey]) {
-            return 1;
-        } else {
-            return 0;
-        }
+        return [[NSUserDefaults standardUserDefaults] boolForKey:nskey] != FALSE;
```

Comment:
Prefer `==` over `!=`.
Prefer `YES`/`NO` over `TRUE`/`FALSE`? Where do these come from?
https://developer.apple.com/documentation/objectivec/objective-c_runtime/boolean_values?language=objc
```suggestion
        return [[NSUserDefaults standardUserDefaults] boolForKey:nskey] == YES;
```

Reply:
Quite right, thanks. TRUE/FALSE work just as well but are not correct. I think they are a hangover from the old Carbon "Boolean" type, and are designed to be interoperable.